lib/screen/onboarding/store/on_boarding_store.dart

```diff
@@ -57,6 +59,35 @@ abstract class _OnBoardingStore with Store {
 
   @observable
   bool isLoading = false;
+  static const List<String> scopes = <String>[
+    'email',
+    'https://www.googleapis.com/auth/contacts.readonly',
+  ];
```

Comment:
```suggestion
```

This scope is unnecessary for our current needs (public info), so let's remove it to streamline permissions and enhance user privacy.